src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml.cs

```diff
@@ -15,5 +19,47 @@ public PackageItemControl()
         {
             InitializeComponent();
         }
+
+        // Whenever the checkbox in the item is checked, the containing item will raise a 
+        // UIA event to convey the fact that the toggle state of the item has changed. The 
+        // event must be raised regardless of whether the toggle state of the item changed 
+        // in response to keyboard, mouse, or programmatic input.
+        private void CheckBox_Checked(object sender, RoutedEventArgs e)
+        {
+            var itemCheckBox = sender as CheckBox;
+            if (itemCheckBox is null || itemCheckBox.Visibility != Visibility.Visible)
+            {
+                return;
+            }
+
+            var parentContentPresenter = VisualTreeHelper.GetParent(this) as ContentPresenter;
```

Comment:
Why do we need to walk the tree to ensure each of the parent controls are not null? I see we're doing that for the ContentPresenter, Border, and ListBoxItem.

Reply:
Walking the tree here because it needs the `ListBoxItem`.